### PR TITLE
German translation of keyboard

### PR DESCRIPTION
**Pull request by boris779**
### Description
German translation for keyboards

### Motivation and Context
Missing german translation for that page

### Types of changes
- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist
- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/webdriver/keyboard.de.md [hunk 1]
```diff
@@ -3,19 +3,16 @@ title: "Tastatur"
...
-{{% notice info %}}
-<i class="fas fa-language"></i> Diese Seite wird von Englisch 
-auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite 
-zu übersetzen indem Du uns einen Pull Reqeust schickst!
- {{% /notice %}}
-Keyboard represents a KeyBoard event. KeyBoard actions are performed by using low-level
-interface which allows us to provide virtualized device input to the web browser.
+Tastatur repräsentiert Tastaturaktionen. Tastatur "Actions" werden mit Hilfe
+eines low-level Interface durchgeführt, dass ein virtualisiertes Eingabegerät
+für den Webbrowser zur Verfügung stellt.
```
Comment (harsha509) [inline docs_source_files/content/webdriver/keyboard.de.md:8]: can we have space in 'Webbrowser'

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @boris779,

Thanks for the PR.

Requested for a minor change

File: docs_source_files/content/webdriver/keyboard.de.md (hunk 1 above)
Comment (boris779) [reply docs_source_files/content/webdriver/keyboard.de.md:8]: Webbrowser is correct as it is in German (without space)


See here: https://www.duden.de/rechtschreibung/Webbrowser

**Review by harsha509 — APPROVED** [review]
Thank you @boris779 !

**Pull request merged**